zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ImageServiceImpl.java

```diff
@@ -32,9 +36,25 @@ public ImageServiceImpl(ImageClient imageClient) {
     public void publishImagesInCollection(Collection collection) throws IOException, ImageAPIException {
 
         Images images = imageClient.getImages(collection.getId());
+        info().data("publishing", true).data("collectionId", collection.getId())
+                .data("numberOfImages", images.getTotalCount())
+                .log("publishing images in collection");
 
         for (Image image : images.getItems()) {
-            imageClient.publishImage(image.getId());
+            switch (image.getState()) {
+                case STATE_CREATED:
+                case STATE_DELETED:
+                    info().data("publishing", true).data("collectionId", collection.getId())
+                            .data("imageId", image.getId())
+                            .data("imageState",image.getState())
+                            .log("skipping publish of image");
+                    break;
+                default:
+                    info().data("publishing", true).data("collectionId", collection.getId())
+                            .data("imageId", image.getId())
+                            .log("publishing image");
+                    imageClient.publishImage(image.getId());
+            }
```

Comment:
Maybe just a preference of mine maybe, but could this be refactored to its own private method?
Happy for this to be ignored and happy to approve otherwise.